transactions/generic_transfer_with_address.cdc

```diff
@@ -31,6 +44,23 @@ transaction(amount: UFix64, to: Address, contractAddress: Address, contractName:
 			?? panic("Could not borrow reference to the owner's Vault!")
 
         self.tempVault <- vaultRef.withdraw(amount: amount)
+
+        // Get the string representation of the address without the 0x
+        var addressString = contractAddress.toString()
+        if addressString.length == 18 {
+            addressString = addressString.slice(from: 2, upTo: 18)
+        }
+        let typeString: String = "A.".concat(addressString).concat(".").concat(contractName).concat(".Vault")
```

Comment:
This covers all existing `Vault` implementations but wouldn't cover multi-vault FT contracts. We should probably note that in the docs and add a `generic_transfer_with_identifier` as was suggested for bridge transactions

Reply:
good point! I created an issue here:https://github.com/onflow/flow-ft/issues/168